powerauth-java-cmd-lib/src/main/java/io/getlime/security/powerauth/lib/cmd/util/ConfigurationUtil.java

```diff
@@ -46,7 +42,7 @@ public static String getApplicationKey(JSONObject clientConfigObject) {
         if (clientConfigObject.get("applicationKey") != null) {
             return (String) clientConfigObject.get("applicationKey");
         } else {
-            return expectedApplicationKey;
+            return null;
```

Comment:
This code can be simplified - the `if-else` is now probably redundant... ? This is in all situations...

Reply:
Fixed.